parse_pom.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# Common utilities used by tools for automating tagging and release
+#
+
+#We want any failures to be fatal
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+REMOTE=origin
+FORCE=
+DRYRUN=
+IGNORE=
+RETAG=
+VERBOSE=
+VERSIONFILE=pom.xml
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+  key="$1"
+  case $key in
+    -f|--force)
+    FORCE=-f
+    shift
+    ;;
+    -i|--ignore)
+    IGNORE=$1
+    shift
+    ;;
+    -v|--verbose)
+    VERBOSE=$1
+    shift
+    ;;
+    --retag)
+    RETAG=$1
+    shift
+    ;;
+    -d|--dryrun)
+    DRYRUN=$1
+    shift
+    ;;
+    -r|--remote)
+    if [ $# -eq 1 ] ; then
+      echo "$1 option requires an argument"
+      exit 2
+    fi
+    REMOTE="$2"
+    shift 
+    shift
+    ;;
+    *)    # unknown option
+    POSITIONAL+=("$1") # save it in an array for later
+    shift # past argument
+    ;;
+  esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+if [ "$#" -ge 1 ] ; then
+  VERSIONFILE=$1
+  shift 1
+fi
+if [ ! -f $VERSIONFILE ]; then
+  echo "File $VERSIONFILE not found"
+  exit 2
+fi
+
+GIT_BRANCH=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD)
+
+function parse_pom()
+{
+  POM_PROJECT=$(mvn help:evaluate -Dexpression=project.artifactId -q -DforceStdout)
```

Comment:
let's specify this mvn dep on the file header

Reply:
Added at start of file, and a test for 'which mvn'